GOVERNANCE.md

```diff
@@ -2,94 +2,171 @@
-- Nominees are expected to be respectful of every community member and to work collaboratively in the spirit of inclusion.
-- Have submitted a minimum of 10 qualifying significant contributions (see list above).
-- You can be nominated by any existing Contributor (L2 or above).
-- Once nominated, there will be a vote by existing Contributors (L3 or above) (see [voting rules](#voting)).
+#### Nomination
 
-It is important to recognize that this role is a privilege, not a right. That privilege must be earned and once earned it can be removed (in a vote by project Stewards). However, under normal circumstances this role exists for as long as the Contributor wishes to continue engaging with the project.
+To be nominated, a nominee is expected to already be performing some of the responsibilities of a Maintainer over the course of at least a couple of weeks. In the past, we have used "10 PRs" as a rough minimum for potential Maintainers, but there is no hard requirement.
 
-Inactive Contributors will have voting rights removed after a certain period of time, however they will always retain their status. Inactivity requirements will be specified in a later governance change.
+In some rare cases, this role may be revoked by a project Steward. However, under normal circumstances this role is granted for as long as the contributor wishes to engage with the project.
 
-### Contributor L3 (Core Contributor)
+#### Nomination Process
 
-Contributor L3 (Core Contributors) are community members who have contributed a significant amount of time to the project through triaging of issues, fixing bugs, implementing enhancements/features, and are trusted community leaders.
+- You can be nominated by any existing Maintainer (L2 or above).
+- Once nominated, there will be a vote by existing Maintainers (L2 and above).
+- See [vote rules & requirements](#voting) for info on how the vote works.
 
-**Responsibilities:**
+### Level 3 (L3) - Core Maintainer
 
-- May merge external pull requests for accepted issues upon reviewing and approving the changes.
-- May merge their own pull requests once they have collected the feedback and approvals they deem necessary.
-  - Caveat: No pull request should be merged without at least one Contributor (L2 or above) comment stating they've looked at the code.
+**Core Maintainers** are community members who have contributed a significant amount of time and energy to the project through issues, bug fixes, implementing enhancements/features, and engagement with the community. A Core Maintainer is considered a trusted leader within the community.
 
-**Nomination:**
+A Core Maintainer has significant sway in software design decisions. For this reason, coding experience is critical for this role. Core Maintainer is the only level of contributor that does require a significant contribution history on GitHub.
```

Comment:
+1 for this sentiment. I love repeating that code isn’t necessary in most roles, because it’s true